x-pack/plugins/fleet/server/services/epm/packages/get.ts

```diff
@@ -346,36 +354,65 @@ export async function getInstalledPackageManifests(
   savedObjectsClient: SavedObjectsClientContract,
   installedPackages: InstalledPackage[]
 ) {
-  const pathFilters = installedPackages.map((installedPackage) => {
+  // Integration manifests
+  const topLevelManifestPathFilters = installedPackages.map((installedPackage) => {
     const { name, version } = installedPackage;
     return nodeBuilder.is(
       `${ASSETS_SAVED_OBJECT_TYPE}.attributes.asset_path`,
       `${name}-${version}/manifest.yml`
     );
   });
 
-  const result = await savedObjectsClient.find<PackageAsset>({
+  const topLevelIntegrationManifests = await savedObjectsClient.find<PackageAsset>({
```

Comment:
**question:** Both this search and the one for dataStreams manifests depend on the `installedPackage` for the nodeBuilder condition. Would be possible to either:
- Combine the condition and perform a single search for the integration manifests and its dataStreams manifests at once.
- Parallelize the 2 SO search, as they are independent.
Currently, the searches are sequential, if we can get a single search or parallelize them it should, ideally, gain something on performance, although It would need to be measured anyway.